src/action.cpp

```diff
@@ -683,6 +685,10 @@ action_id handle_action_menu()
         action_weightings[ACTION_TOGGLE_RUN] = 300;
     }
     // If we're already crouching, make it simple to toggle crouching to off.
```

Comment:
You presumably copied that comment from below. It makes little sense here (it's talking about crouching).